### PR TITLE
Move Phase2 wf in the short matrix to D77

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
As title said, we move PR test workflows of Phase-2 to D77 following the green light of PdmV, to move the D77 relvals from CMSSW_12_1_0_pre5. I use this chance also to define missing PU200 D76, in case it will be used for any testing in PR test.

https://its.cern.ch/jira/projects/PDMVRELVALS/issues/PDMVRELVALS-139

#### PR validation:
-

#### if this PR is a backport please specify the original PR and why you need to backport that PR:
No need of backport.

### CODE REVIEW

**Comment by srimanob** [comment]
@cms-sw/pdmv-l2 Could you please consider?

Note that, you can use https://github.com/cms-sw/cmssw/pulls?q=is%3Apr+label%3Apdmv-pending+is%3Aopen to follow what is pending for PdmV.

**Comment by bbilin** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_2026.py [hunk 1]
```diff
@@ -24,9 +24,11 @@
 numWFIB.extend([31434.0]) #2026D68
 numWFIB.extend([32234.0]) #2026D70
 numWFIB.extend([34634.0]) #2026D76
-numWFIB.extend([34834.999]) #2026D76 premixing combined stage1+stage2 (ttbar+PU50 for PR test)
+numWFIB.extend([34834.99,34834.999]) #2026D76 premixing combined stage1+stage2 (ttbar+PU200, ttbar+PU50 for PR test)
 numWFIB.extend([34634.21,34834.21,34834.9921]) #2026D76 prodlike, prodlike PU, prodlike premix stage1+stage2
 numWFIB.extend([35034.0]) #2026D77
+numWFIB.extend([35234.99,35234.999]) #2026D77 premixing combined stage1+stage2 (ttbar+PU200, ttbar+PU50 for PR test)
```
Comment (perrotta) [inline Configuration/PyReleaseValidation/python/relval_2026.py:30]: (In the next iteration) one could reorder in ascending wf number

**Comment by perrotta** [comment]
+1

**Pull request merged**